src/types.ts

```diff
@@ -1312,6 +1393,32 @@ export type InstanceInfoSerialized =
     | InstanceInfoSerializedV0
     | InstanceInfoSerializedV1;
 
+export interface ContractContext {
+    invoker?: ContractAddress | AccountAddress;
+    contract: ContractAddress;
+    amount: GtuAmount;
```

Comment:
`amount` should be optional as the node will default the value to `0` if not provided.